packages/node_modules/@ciscospark/react-component-utils/src/ReplaceIndexWithBlobUrl.js

```diff
@@ -0,0 +1,31 @@
+const urlTypes = ['url', 'backgroundImage', 'iconUrl'];
+
+/**
+ * replaces the index with the uri in cards
+ * @param {Object} cardsActivity
+ * @param {Array} decryptedURLs
+ * @returns {Object}
+ */
+export function replaceIndexWithBlobURL(cardsActivity, decryptedURLs) {
```

Comment:
If `cardsActivity` is an object with properties and `decryptedURLs` is an array of strings, how do you know what URL corresponds to what object/property?

Reply:
storing the blob URL at the same index as what the URL value of the card object is pointing to